neopixel.py

```diff
@@ -32,6 +32,11 @@
 
 import digitalio
 from neopixel_write import neopixel_write
+try:
+    from pixelbuf import PixelBuf
```

Comment:
How about renaming the C version _pixelbuf and making a Python implementation as well. That way this can be:
```python
try:
    from _pixelbuf import PixelBuf
except ImportError:
    from pixelbuf import PixelBuf
```

That way you only include the Python version of the pixelbuf code as needed.

Reply:
Hmm... So would the end result be a PixelBuf library (both Py and C) that has the majority of the code that NeoPixel and the DotStar libs share in it, and they both use that?  NeoPixel and Dotstar libs then become quite a bit smaller.

Reply:
Yup! That'd be ideal. It can be used by WS2801 and Pixie too.